test/functional/services/common/test_subjects.ts

```diff
@@ -360,11 +360,12 @@ export class TestSubjects extends FtrService {
     await this.findService.waitForElementHidden(element, timeout);
   }
 
-  public async waitForEnabled(selector: string, timeout: number = this.TRY_TIME): Promise<void> {
-    await this.retry.tryForTime(timeout, async () => {
+  public async waitForEnabled(selector: string, timeout: number = this.TRY_TIME): Promise<boolean> {
+    const success = await this.retry.tryForTime(timeout, async () => {
       const element = await this.find(selector);
       return (await element.isDisplayed()) && (await element.isEnabled());
     });
+    return success;
```

Comment:
Returning `success` here so that the consumers can choose whether or not they want to continue with the remainder of their test if the element does not got enabled in the given time period.